recipes/libxlsxwriter/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.5)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(example example.cpp)
```

```suggestion
add_executable(example example.c)
```

Change it to a C file, as libxlsxwriter is a C pacakge.